types/key/key.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package key defines some types related to curve25519 keys.
+package key
+
+import "golang.org/x/crypto/curve25519"
```

Comment:
I don't think it's worth avoiding this dep. It's coming along for the ride one way or another in tailscale binaries.